src/components/Buttons.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Proptypes from "prop-types";
+
+class Buttons extends React.Component {
+  static proptypes = {
```

Comment:
Change to `propTypes`, plz 😉

Reply:
No hacia falta los PropTypes en mes, asi que lo borramos